SpokojDucha/components/SearchDecedent.js

```diff
@@ -64,98 +58,29 @@ const SearchDecedent = () => {
-    if (isNaN(id)) {
-      Alert.alert('Invalid ID', 'The decedent ID is not valid.');
-      return;
-    }
-    navigation.navigate('UploadVideo', { id });
-  };
-
-  const renderItem = ({ item }) => (
-    <View style={styles.resultItem}>
-      {item.imageBase64 && (
-        <TouchableOpacity onPress={() => setSelectedImage(item.imageBase64)}>
-          <Image
-            source={{ uri: `data:image/png;base64,${item.imageBase64}` }}
-            style={styles.image}
-          />
-        </TouchableOpacity>
-      )}
-      <View style={styles.textContainer}>
-        <Text style={[styles.boldText, { fontSize: 16 + fontSizeDelta }]}>{item.name} {item.surname}</Text>
-        <Text style={[styles.resultText, { fontSize: 16 + fontSizeDelta }]}>Data urodzenia: {formatDate(item.birthDate)}</Text>
-        <Text style={[styles.resultText, { fontSize: 16 + fontSizeDelta }]}>Data śmierci: {formatDate(item.deathDate)}</Text>
-        <Text style={[styles.resultText, { fontSize: 16 + fontSizeDelta }]}>Opis: {item.description}</Text>
-        <Text style={[styles.resultText, { fontSize: 16 + fontSizeDelta }]}>Miasto: {item.city}</Text>
-        <View style={styles.buttonContainer}>
-          <TouchableOpacity style={styles.button} onPress={() => handleNavigate(item.id)}>
-            <Text style={[styles.buttonText, { fontSize: 16 + fontSizeDelta }]}>Nawiguj</Text>
-          </TouchableOpacity>
-          <TouchableOpacity style={styles.button} onPress={() => handleUploadVideo(item.id)}>
-            <Text style={[styles.buttonText, { fontSize: 16 + fontSizeDelta }]}>Dodaj film</Text>
-          </TouchableOpacity>
-        </View>
-      </View>
-    </View>
-  );
-
   return (
     <View style={styles.container}>
       <TextInput
         placeholder="Name"
-        style={[styles.input, { fontSize: 16 + fontSizeDelta }]}
+        style={[styles.input, { fontSize: 16 + fontSizeDelta, paddingVertical: 10 + fontSizeDelta * 0.25 }]} 
         onChangeText={value => handleInputChange('name', value)}
         value={keywords.name}
       />
       <TextInput
         placeholder="Surname"
-        style={[styles.input, { fontSize: 16 + fontSizeDelta }]}
+        style={[styles.input, { fontSize: 16 + fontSizeDelta, paddingVertical: 10 + fontSizeDelta * 0.25 }]}
         onChangeText={value => handleInputChange('surname', value)}
         value={keywords.surname}
       />
       <TextInput
         placeholder="City"
-        style={[styles.input, { fontSize: 16 + fontSizeDelta }]}
+        style={[styles.input, { fontSize: 16 + fontSizeDelta, paddingVertical: 10 + fontSizeDelta * 0.25 }]}
         onChangeText={value => handleInputChange('city', value)}
         value={keywords.city}
       />
       <TouchableOpacity style={styles.searchButton} onPress={handleSearch}>
         <Text style={[styles.buttonText, { fontSize: 16 + fontSizeDelta }]}>Szukaj</Text>
       </TouchableOpacity>
-      {results.length > 0 && (
```

Comment:
Another part responsible for the search handling. And also needed for the Model that was made our objective for this sprint